### PR TITLE
[docs]upgraded the Debezium doc version to 1.5.2.Final.

### PR DESCRIPTION
**Pull request by GOODBOY008**
Fix [issue-322](https://github.com/ververica/flink-cdc-connectors/issues/322).
Summary:
Upgrade the debezium doc version to 1.5.

### CODE REVIEW

**Comment by GOODBOY008** [comment]
Hi,@leonardBang .
Please code review,Thank you!

**Comment by leonardBang** [comment]
@GOODBOY008 Thanks for the contribution, the change look good generally, but the code style is problematic,  you need set
you IDE project code style using  the file under `tools/maven/checkstyle.xml`, and you can also run `maven spotless:apply` to  format your change before commit.

**Comment by GOODBOY008** [comment]
> @GOODBOY008 Thanks for the contribution, the change look good generally, but the code style is problematic, you need set
> you IDE project code style using the file under `tools/maven/checkstyle.xml`, and you can also run `maven spotless:apply` to format your change before commit.

**Comment by GOODBOY008** [comment]
@leonardBang Check Env works incorrectly.I had check style use **maven spotless:apply** ,but nothing happened.
image

File: docs/content/connectors/mysql-cdc.md [hunk 1]
```diff
@@ -242,7 +242,7 @@ During a snapshot operation, the connector will query each included table to pro
       <td>String</td>
       <td>Pass-through Debezium's properties to Debezium Embedded Engine which is used to capture data changes from MySQL server.
           For example: <code>'debezium.snapshot.mode' = 'never'</code>.
-          See more about the <a href="https://debezium.io/documentation/reference/1.2/connectors/mysql.html#mysql-connector-configuration-properties_debezium">Debezium's MySQL Connector properties</a></td> 
+          See more about the <a href="https://debezium.io/documentation/reference/1.5/connectors/mysql.html#_required_debezium_mysql_connector_configuration_properties">Debezium's MySQL Connector properties</a></td>
```
Comment (leonardBang) [inline docs/content/connectors/mysql-cdc.md:245]:
```suggestion
          See more about the <a href="https://debezium.io/documentation/reference/1.5/connectors/mysql.html#mysql-connector-properties">Debezium's MySQL Connector properties</a></td> 
```

**Review by leonardBang — COMMENTED** [review]
Thanks @GOODBOY008 for the contribution, the change looks good, I only left one minor comment.
Could you also open a PR for release-2.0 branch ?

**Comment by GOODBOY008** [comment]
> Thanks @GOODBOY008 for the contribution, the change looks good, I only left one minor comment.
> Could you also open a PR for release-2.0 branch ?

@leonardBang ,I will open  a PR for release-2.0 branch.Thank you~

**Review by leonardBang — APPROVED** [review]
LGTM

**Pull request merged**